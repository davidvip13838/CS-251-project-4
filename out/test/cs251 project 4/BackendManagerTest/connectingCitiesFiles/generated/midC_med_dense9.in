500
86 82
61 33
2 86
68 32
41 23
60 64
63 68
37 40
10 82
4 100
32 89
1 10
17 21
92 57
70 79
90 12
44 16
16 35
99 67
93 66
60 73
37 43
22 4
68 33
6 29
24 76
70 94
92 12
56 100
33 14
71 92
79 98
49 74
37 90
87 58
8 74
7 49
71 44
13 50
46 53
16 48
6 52
28 99
49 94
42 53
40 60
16 93
34 15
41 25
3 94
24 32
44 34
11 37
61 56
99 93
49 22
6 39
58 17
21 4
0 89
70 69
47 15
36 24
60 89
54 6
32 61
92 24
66 90
7 7
71 91
38 34
75 72
99 37
42 18
82 63
78 58
33 54
5 44
26 54
2 15
1 67
58 55
44 93
37 28
68 80
60 25
59 66
29 0
14 52
41 35
97 48
20 86
14 93
31 21
99 7
9 86
6 95
21 50
40 39
64 83
68 57
21 74
82 10
30 45
41 50
14 74
25 17
87 44
7 37
13 41
97 14
95 31
91 90
36 16
97 56
77 18
46 87
6 55
76 77
47 60
48 67
19 57
19 45
21 52
40 59
70 28
6 91
31 0
77 33
85 83
49 19
39 25
33 35
26 81
50 33
97 51
54 87
40 10
81 100
3 54
22 21
41 17
94 69
65 1
35 12
45 97
45 21
2 85
39 93
22 71
46 51
17 64
86 77
53 87
13 54
11 98
55 73
0 44
90 7
99 81
58 68
86 29
100 72
2 17
16 14
38 31
25 91
10 76
20 41
56 98
20 36
81 89
52 68
32 72
9 90
77 12
33 6
66 89
15 60
85 81
26 46
34 30
24 99
51 8
100 79
26 66
39 80
79 33
25 7
76 92
85 51
63 86
73 39
47 97
89 67
67 74
30 36
32 66
97 90
29 55
12 4
46 68
47 69
4 53
22 42
39 29
94 39
65 67
61 31
20 18
41 61
7 81
74 79
99 80
67 61
71 54
58 84
53 37
28 59
57 26
30 4
93 43
24 37
35 22
45 72
17 16
92 15
7 90
32 17
63 27
44 24
93 64
89 19
71 76
18 82
52 61
12 40
52 32
97 35
17 20
25 62
64 34
59 30
48 40
18 49
66 27
25 26
63 99
61 44
90 73
80 57
55 99
61 43
63 47
81 61
37 92
3 56
26 2
72 85
67 58
8 3
37 12
21 95
20 68
74 92
65 28
71 74
43 80
36 2
67 30
69 35
16 47
100 95
36 17
17 46
38 63
13 33
53 34
10 1
37 54
36 81
13 22
15 25
48 24
41 96
98 94
36 91
73 6
20 6
46 5
76 27
58 96
56 77
87 66
79 64
59 37
100 49
37 27
16 79
38 25
3 4
40 93
32 80
5 71
3 41
62 18
55 92
6 92
66 21
57 24
92 73
88 13
30 93
50 89
96 57
9 39
94 2
91 45
59 44
81 49
98 88
6 80
91 31
35 46
30 42
34 24
81 62
99 95
79 79
2 55
6 16
43 52
88 21
52 27
69 22
81 22
24 31
24 82
43 82
16 28
7 44
6 10
20 62
74 26
28 41
63 64
20 31
52 41
66 28
9 23
38 71
36 73
1 94
84 38
35 53
53 11
4 73
64 58
38 32
83 13
19 93
4 12
15 15
79 17
44 85
80 69
41 95
54 1
87 80
50 14
94 40
43 96
33 38
100 31
5 22
8 54
84 23
46 57
97 41
86 69
38 46
30 49
90 80
35 49
13 34
33 65
2 59
20 10
11 31
87 4
56 60
96 95
92 83
64 95
22 7
15 40
26 24
20 91
21 60
70 39
85 59
28 85
88 35
93 97
37 30
34 94
62 4
33 83
48 6
67 82
61 75
91 96
15 78
37 42
52 71
39 48
91 1
5 89
70 83
40 74
22 15
67 83
81 28
75 27
42 23
85 20
74 94
31 90
56 33
87 55
89 49
24 79
71 40
51 72
41 19
76 88
39 14
15 47
82 45
10 50
83 30
18 13
71 33
45 42
78 13
24 39
11 15
36 31
83 54
82 100
19 10
9 66
59 100
69 71
87 8
14 48
12 11
85 98
67 25
51 92
9 83
80 63
75 58
56 87
72 60
83 14
37 37
9 96
12 92
57 43
49 32
96 55
78 48
40 94
15 85
36 74
97 9
61 40
77 55
93 11
88 94
92 27
97 19
57 83
63 51
7 96
14 54
0 24
89 41
66 31
8 63
28 48
99 12
23 47
84 90
46 45
61 88
50 49
7 38
70 26
