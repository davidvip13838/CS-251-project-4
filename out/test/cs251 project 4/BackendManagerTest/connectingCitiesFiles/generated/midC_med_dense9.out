1533.3248824513958